docs/conf.py

```diff
@@ -398,3 +398,19 @@ def setup(app):
     f.write(zenodo_record)
 
 print(zenodo_record)
+
+
+# -- Creating tutorials.rst ---------------------------------------------------
+
+notebooks = ""
+
+for root, dirs, fnames in os.walk("io/"):
+    for fname in fnames:
+        if fname.endswith(".ipynb") and "checkpoint" not in fname:
+            notebooks += "\n* :doc:`" + root + "/" + fname[:-6] + "`"
+
+title = "Tutorials\n*********\n"
+description = "\nThe following pages contain the TARDIS tutorials:\n"
+
+with open("tutorials.rst", mode="wt", encoding="utf-8") as f:
+    f.write(title + description + notebooks)
```

Comment:
You can do something like this (provided you remove leading & trailing `\n` from `tiltle` and `description`):
```suggestion
    f.write(f"{title}\n\n{description}\n{notebooks})
```